code/modules/mining/lavaland/megafauna_loot.dm

```diff
@@ -749,7 +749,7 @@
 				"wings" = "None",
 				"frills" = "None",
 				"spines" = "Long",
-				"body_markings" = "Dark Tiger Body",
+				"lizard_markings" = "Dark Tiger Body",
```

Comment:
You don't have to do it in this pr but we really should go around and replace this kinda stuff with
`/datum/sprite_accessory/lizard_markings/dtiger::name`

Reply:
that's be a really good idea, all of this code is filled with hardcoded strings like this

Reply:
that is SUCH a good idea lets go